### PR TITLE
Newformat

### PR DESCRIPTION
**Pull request by agnivade**

### CODE REVIEW

File: test/parser.spec.js [hunk 1]
```diff
@@ -41,9 +41,9 @@ describe('Parser', function() {
       '\n# tar' +
       '\n> archiving utility' +
       '\n' +
-      '\n- create an archive' +
+      '\ncreate an archive' +
```
Comment (waldyrious) [inline test/parser.spec.js]: Shouldn't the first letter be capitalized?

Comment (agnivade) [inline test/parser.spec.js]: I think that's the work of the linter. The client is not bothered about it. It will display verbatim.

Comment (waldyrious) [inline test/parser.spec.js]: Sure, I understand that it doesn't care, but since this code is meant to represent real tldr-pages content, it would make sense IMO to match the actual format, for consistency. But of course, that change is not directly related to this PR, so it shouldn't hold up merging.

**Comment by agnivade** [comment]
This is unlikely to happen in the near future. Let's just close it and we can revisit this if we start to work on it again.
